apps/vaporgui/NavigationEventRouter.cpp

```diff
@@ -289,11 +289,9 @@ void NavigationEventRouter::_performAutoStretching(string dataSetName) {
 
         // If a dimension's scale is not 1.f, the user has saved a session with
         // a non-default value.  Don't modify it.
-        if ( scales[ xDimension ] != 1.f )
-            continue;
-        if ( scales[ yDimension ] != 1.f )
-            continue;
-        if ( scales[ zDimension ] != 1.f )
+        if ( scales[ xDimension ] != 1.f &&
+             scales[ yDimension ] != 1.f &&
+             scales[ zDimension ] != 1.f )
```

Comment:
Same logic issue as in PR #1539. Not sure why these same changes are appearing in two PRs....

Reply:
Right.  This is because I needed to merge this branch with #1539 for testing.

Both this branch and #1539 contain the same commit that applies these changes, so there shouldn't be any conflicts.

Reply:
It turns out that this was correct the first time.  I'll remove it from this branch.